frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -42,6 +42,14 @@ const ReviewPage = () => {
   const [title, setTitle] = useState<string>("");
   const [language, setLanguage] = useState<string>("");
 
+  const [isCommenting, setIsCommenting] = useState(false);
```

Comment:
hmmm perhaps instead of managing `isCommenting` and `commentLine`, we can use only `commentLine`. `commentLine == -1` can be equivalent to 'there is no comment in WIP'. Thoughts?

Reply:
`storyTranslationContentId` could be used in place of isCommenting too but i could be wrong, In either case, the `commentLine` or `storyTranslationContentId` would need to be reset after a work-in-progress comment is created and the WIPComment component is collapsed